src/main/java/fr/moribus/imageonmap/commands/maptool/NewCommand.java

```diff
@@ -92,12 +92,35 @@ protected void run() throws CommandException {
         }
 
         if (args.length >= 2) {
-            if (args.length >= 4) {
-                width = Integer.parseInt(args[2]);
-                height = Integer.parseInt(args[3]);
+            if (args.length >= 3) {
+                try {
+                    if (args.length >= 4) {
+                        width = Integer.parseInt(args[2]);
+                        height = Integer.parseInt(args[3]);
+                    } else {
+                        String[] size;
```

Comment:
I used String[] but it may be improved by using  `Pair<String> size` and `size.a size.b`, we may add one in QuartzLib (because there are too many implementation of this)